core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -377,8 +377,50 @@ static function (array $value): array {
                     ->defaultNull()
                 ->end()
                 ->arrayNode('valid_extensions')
+                    ->info('Adds, removes or overwrites the list of enabled image extensions that can be used.')
                     ->prototype('scalar')->end()
                     ->defaultValue(['jpg', 'jpeg', 'gif', 'png', 'tif', 'tiff', 'bmp', 'svg', 'svgz', 'webp', 'avif'])
+                    ->example(['+heic', '-svgz', 'jxl'])
+                    ->validate()
+                        ->ifTrue(
+                            static function (array $extensions): bool {
+                                foreach ($extensions as $extension) {
+                                    if (!preg_match('/^[+-]?.+$/', $extension)) {
```

Comment:
```suggestion
                                    if (!preg_match('/^[+-]?[a-z0-9]+$/', $extension)) {
```

Would it make sense to restrict this further? Otherwise values like `\\\` would be considered valid, too.